chainguard-dev/edu
# Remove Inky references in Comic

## Type of change
Removes Inky references in the comic section of the Academy, replacing with Linky

### Why are we making this change?
Brings the Academy in-line with branding refresh

### What are the acceptance criteria? 
All Inky references are removed and replaced with Linky where appropriate

### How should this PR be tested?
Check the comics section of the deploy preview and scan for Linky vs Inky changes. Direct link: https://deploy-preview-1761--ornate-narwhal-088216.netlify.app/software-security/comics/
